README.md

```diff
@@ -22,20 +22,24 @@ Get the code:
 To build for the EV3:
 
 *   Install [Docker] (requires Linux/macOS 10.10.3+/Window 10 Pro)
+*   Install build dependancies
```

Comment:
* will add some text about `qemu-user-static` being only for linux
* libgrx here got rid of the `No package 'grx-3.0' found` when doing `cmake <path-to-brickdm-source> -DCMAKE_BUILD_TYPE=string:Debug -DBRICKMAN_TEST=bool:Yes`.  Was that a false positive?

Still not sure how to ev3devkit installed so the cmake will work...

Reply:
This part of the readme is "To build for the EV3". Since that build runs in a docker container, I don't see how it could make a difference here.

If you are building Brickman targeting the host machine architecture instead of the EV3, then I would expect that you need to install all of the build dependencies (but not qemu-user-static). But that should be under a different heading.

Reply:
ok I follow you now, will move the grx part down to the "To build the desktop test" section.  How can I get ev3devkit installed?  I see that it is installed in the docker container but I am not sure where it came from originally so that I can go get it.

Reply:
>  How can I get ev3devkit installed?

For now, you have to build it yourself. We need to add it to the PPA.

> To build the desktop test

Tip: 

    sudo apt-add-repository -u -s  ppa:ev3dev/tools
    sudo apt build-dep brickman

Other than the missing package (which we can fix), this should get the job done in as concisely as possible.

Reply:
`sudo apt build-dep brickman` fails for me due to `E: Unable to find a source package for brickman` but I can do `sudo mk-build-deps --install debian/control`